src/action.c

```diff
@@ -179,7 +183,14 @@ action_arg_from_xml_node(struct action *action, char *nodename, char *content)
 		}
 		break;
 	case ACTION_TYPE_MOVE_TO_EDGE:
+		if (!strcmp(argument, "snap")) {
+			action_arg_add_str(action, argument, content);
+			goto cleanup;
+		}
+		/* Falls through */
```

Comment:
Similar here to the `view_edge` enum, I think we should use a bool argument here. Thus parsing it here instead of all over again later in the code.